src/js/trove/world-lib.js

```diff
@@ -604,8 +667,13 @@ define(["js/runtime-util"], function(util) {
         this.fail = fail;
     }
 
-    BigBangRecord.prototype.restart = function() {
-        bigBang(this.top, this.world, this.handlerCreators, this.attribs);
+    BigBangRecord.prototype.start = function() {
```

Comment:
I actually like the name `restart` better for this, FWIW, because it is called multiple times per object, and complements calls to `pause`.